### PR TITLE
Fix golint warnings for publisher package

### PR DESCRIPTION
**Pull request by ShajithaMohammed**
As part of #58

---
This change is [Reviewable](https://reviewable.io/reviews/fission/fission/1822)

### CODE REVIEW

File: pkg/publisher/publisher.go [hunk 1]
```diff
@@ -17,10 +17,10 @@ limitations under the License.
 package publisher
 
 type (
+	// Publisher publish an request to a "target".  Target depends on the
+	// publisher: it's a URL in the case of a webhook publisher, or a queue
+	// name in a queue-based publisher such as NATS.
```
Comment (therahulbhati) [inline pkg/publisher/publisher.go:22]: The comment on the Publisher interface can be written as:
```
// Publisher interface wraps the Publish method that publishes message and headers
// 
// Publish an request to a "target".  Target's meaning depends on the	
// publisher: it's a URL in the case of a webhook publisher, or a queue	
// name in a queue-based publisher such as NATS.
```

File: pkg/publisher/webhookPublisher.go [hunk 2]
```diff
@@ -47,6 +47,7 @@ type (
 	}
 )
 
+// MakeWebhookPublisher triggers a publish request to the given baseUrl with 10 maximum retries in case of error
```
Comment (therahulbhati) [inline pkg/publisher/webhookPublisher.go:50]:
```suggestion
// MakeWebhookPublisher creates a WebhookPublisher object for the given baseUrl
```

File: pkg/publisher/webhookPublisher.go [hunk 3]
```diff
@@ -60,6 +61,7 @@ func MakeWebhookPublisher(logger *zap.Logger, baseUrl string) *WebhookPublisher
 	return p
 }
 
+// Publish sends a request to the target with given payload with a defined retries in case of error
```
Comment (therahulbhati) [inline pkg/publisher/webhookPublisher.go:64]:
```suggestion
// Publish sends a request to the target with payload having given body and headers
```

**Review by therahulbhati — CHANGES_REQUESTED** [review]

**Review by therahulbhati — APPROVED** [review]

**Review by therahulbhati — APPROVED** [review]
LGTM

**Comment by vishal-biyani** [comment]
Thanks for the PR @ShajithaMohammed

**Comment by chetanpdeshmukh** [comment]
Hello @ShajithaMohammed ,
Thanks again for the PR!

Thank you for contributing to Fission project. Could you please fill out [this form](https://docs.google.com/forms/d/e/1FAIpQLSdhxtWL5N2PU7MYC2IRRAzYw-CrG9QtsYNvkiiOPVEQj8i51w/viewform?usp=sf_link), so we can send you the well deserved awesome swags :)

Team Fission

**Pull request merged**